RevenueCat/Scripts/Purchases.cs

```diff
@@ -177,7 +210,7 @@ public void RestorePurchases(CustomerInfoFunc callback)
         _wrapper.RestorePurchases();
     }
 
-    [Obsolete("Deprecated, use set<NetworkId> methods instead.")]
+    [Obsolete("Deprecated, use set<NetworkId> methods instead.", true)]
     public void AddAttributionData(string dataJson, AttributionNetwork network, string networkUserId = null)
     {
         _wrapper.AddAttributionData((int)network, dataJson, networkUserId);
```

Comment:
This won't be called anymore so should we remove `AddAttributionData` from the wrapper?